### PR TITLE
Bugfix/page number not included in url

### PR DESCRIPTION
**Pull request by dgvives**
solves #13 
Pagination was not working when listing licenses. 
Feel free to take just the fix and ignore the test if you find it noisy

### CODE REVIEW

**Comment by meJevin** [comment]
@dgvives thank you so much for this PR! Especially for writing tests, because I kind of forgot about the at some point 😄

File: Source/Cryptlex.Net/Core/Services/Base/BaseService.cs [hunk 1]
```diff
@@ -28,7 +28,7 @@ private async Task<RequestResult> QueryStringRequest(string uri, object? data =
 
             if (data is not null)
             {
-                Utils.AppendQueryString(uri, data.ToQueryString());
+                uri = Utils.AppendQueryString(uri, data.ToQueryString());
```
Comment (meJevin) [inline Source/Cryptlex.Net/Core/Services/Base/BaseService.cs:31]: Hah, I really missed this one. Good catch 😄

**Review by meJevin — APPROVED** [review]

**Comment by meJevin** [comment]
Just realized that this is actually a really big mistake by me because all GET requests did not have a query string attached to them 😨 
I'll do a quick Nuget update while nobody is looking

**Comment by dgvives** [comment]
Suggestion: Squash and merge. Keep your history as clean as possible
image

**Comment by meJevin** [comment]
Oooh, thanks! I didn't event notice that button there.

**Pull request merged**